lib/SampleService/core/config.py

```diff
@@ -198,7 +222,7 @@ def _check_string_req(s: Optional[str], name: str) -> str:
 }
 
 
-def get_validators(url: str) -> MetadataValidatorSet:
+def get_validators(repo_path: Optional[str] = None, repo_file: Optional[str] = None, prerelease_ok: Optional[bool] = False, url: Optional[str] = None, token: Optional[str] = None) -> MetadataValidatorSet:
```

Comment:
This line is too long...
```suggestion
def get_validators(
        repo_path: Optional[str] = None,
        repo_file: Optional[str] = None,
        prerelease_ok: bool = False,
        url: Optional[str] = None,
        token: Optional[str] = None
        ) -> MetadataValidatorSet:
```

Also there's no reason to make a bool optional unless you want trinary logic, which I don't think you want here

Reply:
:+1: